types/money/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for money 0.2
+// Project: https://github.com/openexchangerates/money.js
+// Definitions by: Ivan Ergunov <https://github.com/hozblok>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace fx {
+    type Currency = string;
+
+    type Rates = Record<string, number | string>;
+
+    type Options = {
+        from?: Currency | null;
+        to?: Currency | null;
+    } | null;
+
+    type Value = string | number | string[] | number[];
+
+    type Result<T_VAL extends Value> = T_VAL extends string | number ? number : number[];
+
+    interface WrapperPrototype<T_VAL extends Value> {
+        convert: {
+            (opts?: Options): Result<T_VAL>;
+        };
+
+        from: {
+            (currency: Currency): Wrapper<T_VAL>;
+        };
+
+        to: {
+            (currency: Currency): Result<T_VAL>;
+        };
+    }
+
+    interface Wrapper<T_VAL extends Value> extends WrapperPrototype<T_VAL> {
+        readonly _v: number;
+        readonly _fx?: Currency;
+    }
+
+    interface WrapperConstructor<T_VAL extends Value> {
+        readonly prototype: WrapperPrototype<T_VAL>;
+        new (val: T_VAL): Wrapper<T_VAL>;
+    }
+}
+
+interface FX {
+    /**
+     * If fx(val) is called as a function, it returns a wrapped object that can be used OO-style
+     */
+    <T_VAL extends fx.Value>(val: T_VAL): fx.Wrapper<T_VAL>;
+    new <T_VAL extends fx.Value>(val: T_VAL): fx.Wrapper<T_VAL>;
+
+    /**
+     * Current version
+     */
+    version: string;
```

Comment:
```suggestion
    version: '0.2';
```
IMO type literal works best here (for versions)

Reply:
What will happen if the version of the library is updated to `'0.2.1'`? And no code changes with respect to types...

Will our user have to search for the required version of the `@types/money`? or put update to `DefinitelyTyped/DefinitelyTyped`? Version `'0.2.1'` in the variable with `'0.2'` type looks confusing.
How do you think?